org.omg.sysml.jupyter.kernel/settings.gradle

```diff
@@ -1,2 +1,18 @@
+//config for local repositories. this enables us to use gradle plugins locally
+//this is a temporary workaround until the java 17 compatible version of the jupyter-kernel plugin is deployed to plugin sites
+buildscript {
+    repositories {
+        mavenCentral()
+        flatDir {
+            dirs 'lib'
```

Comment:
I do not think it's needed because actually the classpath of dependencies of Line 13 is effective.  Also I think we should write this dependency to build.gradle rather than settings.gradle because we do not have any subprojects.   I will make a commit if you're ok.